Library/Homebrew/keg.rb

```diff
@@ -105,7 +105,13 @@ def self.find_some_installed_dependents(kegs)
     #
     # This happens after the initial dependency check because it's sloooow.
     remaining_formulae = Formula.installed.select do |f|
-      f.installed_kegs.any? { |k| Tab.for_keg(k).runtime_dependencies.nil? }
+      installed_kegs = f.installed_kegs
+
+      # All installed kegs are going to be removed anyway,
```

Comment:
Cool, makes sense. I think the comment could be expanded on a bit to make this clearer e.g. why they will be "removed"